### DISCUSSION TITLE
Ergonomics

### DISCUSSION THREAD

**Discussion by beuaaa** [beuaaa/pywinauto_recorder#20] [Q&A]
I wonder about the ergonomics of Pywinauto_recorder, in the next versions I want to add more functions but activating the functions by keyboard shortcuts is not very ergonomic.

I'm thinking of the following evolution:
All functions will be represented by an icon. The icons will be aligned at the top of the screen as currently but with more icons. To activate a function you will have to hover over the icon with the mouse and press the control key twice (like a double mouse click).

Rather than adding a lot of complicated shortcuts. This double control will be the main keyboard action used by Pywinauto_recoder.

What do you think of this idea?
Do you think double control is the best choice? (why not double shift or something else)

**Comment by OnlinePage** [comment]
Superb Idea! Even I found the shortcuts quite inconvenient at first, so adding the button on the top is good decision.
Also add a feature to select the location where user want to save the recorded python files!

**Reply by beuaaa** [reply]
Finally I think it's better to limit the number of icons displayed at the top of the screen in order to leave the maximum space for the applications to be automated. For this reason I added a menu in the taskbar. This tray menu can activate all the functions. 

In particular the function to display the explorer window where the recorded Python files are saved. Do you think this is not enough?

I am glad that you like the idea of the double control. We have to think about it. 
Maybe it could work depending on the current mode like this:

- If current mode is Recording, double CTRL on the recording icon will Stop recording
- If current mode is Pause, double CTRL on the Stop icon will Start recording
- If current mode is Informations, double CTRL on the magnifying glass icon will Stop displaying information

Thank you very much for your opinion it helps me a lot. Feel free to propose your ideas and to criticize mine.
